docs/core/tools/dotnet-nuget-add-source.md

```diff
@@ -38,6 +38,10 @@ The `dotnet nuget add source` command adds a new package source to your NuGet co
 
 [!INCLUDE [configfile](../../../includes/cli-configfile.md)]
 
+- **`--allow-insecure-connections`**
+  
+  Allows HTTP connections for adding or updating packages. This method is not secure.
```

Comment:
I think it would be worthwhile documenting which version of the .NET SDK it's available on. I know it's definitely not in 8.0.100, which is what source-build and therefore distros like Ubuntu and RedHat have. I don't remember if this made it into 8.0.400, or 9.0.100.

Reply:
@zivkan When you have a chance to establish for sure which version it's in, I'll open another PR to add it to the article.

Reply:
This was introduced with the http-errors, 9.0.100